packages/dnb-eufemia/src/components/pagination/PaginationBar.tsx

```diff
@@ -136,16 +136,17 @@ const PaginationBar = (innerProps: PaginationBarProps) => {
       getTranslation(innerProps).Pagination
     )
 
-  const prevIsDisabled = currentPage === 1
-  const nextIsDisabled = currentPage === pageCount || pageCount === 0
+  const prevIsDisabled = currentPage > -1 ? currentPage === 1 : true
```

Comment:
```suggestion
  const prevIsDisabled = currentPage <== 1
```
For faster readability... Think it would act the same (the only difference her is that it will be `true` for currentPage==0, which also should disable previous button. Do you have any inputs?

Reply:
I want `prevIsDisabled` to be true, if an invalid value was given. I don't think `currentPage <== 1` will give us that?

Reply:
Ah, I understand. Then your solution is the better